<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1173</width>
    <height>763</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1173</width>
    <height>763</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(217, 226, 239)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>1041</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);

</string>
    </property>
    <widget class="QWidget" name="userPage">
     <widget class="QTableWidget" name="userlist">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>911</width>
        <height>581</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section{
background:rgb(255, 255, 255);
	color: rgb(0, 170, 255);
	border: none;
}
QScrollBar{background:transparent; height:12px;}
	QScrollBar::handle{background:rgb(224, 248, 255); border:2px solid transparent; border-radius:5px;}
	QScrollBar::handle:hover{background:gray;}
	QScrollBar::sub-line{background:transparent;}
	QScrollBar::add-line{background:transparent;}
QTableWidget::item { background-color: rgb(244, 244, 244); }
QTableWidget::item:nth-child(odd) { 
	background-color: rgb(231, 255, 255); }

QTableWidget { border: none; }</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="rowCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>300</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>账号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>密码</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="deleteList">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="changeList">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="adIDEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adseEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入密码</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FlightPage">
     <widget class="QPushButton" name="cancelButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="flightlist">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>911</width>
        <height>581</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section{
background:rgb(255, 255, 255);
	color: rgb(0, 170, 255);
	border: none;
}
QScrollBar{background:transparent; height:12px;}
	QScrollBar::handle{background:rgb(224, 248, 255); border:2px solid transparent; border-radius:5px;}
	QScrollBar::handle:hover{background:gray;}
	QScrollBar::sub-line{background:transparent;}
	QScrollBar::add-line{background:transparent;}
QTableWidget::item { background-color: rgb(244, 244, 244); }
QTableWidget::item:nth-child(odd) { 
	background-color: rgb(231, 255, 255); }

QTableWidget { border: none; }

</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>147</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>航班编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>航空公司</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发地</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>起飞时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>目的地</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>降落时间</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="changeList_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteList_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="NewList_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
      </property>
      <property name="text">
       <string>新建</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="numEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入航班编号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="companyEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入航空公司</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fromEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入出发地</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入起飞时间</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="arrEdit">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入目的地</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="landEdit">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 8pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入降落时间</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>161</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QPushButton" name="userPageButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>个人信息管理</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="FlightPageButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>航班信息管理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background:rgb(255, 255, 255);
	border-radius:0px;
	color: rgb(0, 170, 255);
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	border:2px solid rgb(0, 170, 255);
	border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="back" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 242, 243);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
